build.fsx

```diff
@@ -431,6 +431,8 @@ let getCurrentAndLastReleaseFromChangelog () =
 
 pipeline "Release" {
     workingDir __SOURCE_DIRECTORY__
+    stage "Build" { run "dotnet build -c Release" }
+    stage "Pack" { run "dotnet pack --no-restore -c Release --tl" }
```

Comment:
(striiiiictly speaking it may be necessary to restore here too - the dependency graph can change in arbitrary ways depending on what's being packed - but I would be surprised if that made a difference for a .NET global tool that is non-runtime-id-specific)